lib/signup.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+
+class Signup extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Visit Africa'),
+        centerTitle: true,
+        backgroundColor: Color(0xffB15C1E),
```

Comment:
Why do we have this `app bar ` here not in the original design? It kills everything. what do you think about removing it?